test/e2e/cheerio-impit-ts/actor/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM node:20 AS builder
+
+COPY /packages ./packages
+COPY /package*.json ./
+COPY /tsconfig.json ./
+COPY /main.ts ./
+RUN npm --quiet set progress=false \
+    && npm install --only=prod --no-optional --no-audit \
+    && npm update \
+    && npm run build
+
+FROM lwthiker/curl-impersonate
```

Comment:
I believe that `node:20` would've been enough? Though if we used node:23, maybe we wouldn't need the typescript build step.